pkg/proxy/proxy.go

```diff
@@ -89,13 +90,13 @@ func (p *Proxy) StartProxy() *http.Server {
 	router := echo.New()
 	router.Logger.SetLevel(glog.INFO)
 	router.HTTPErrorHandler = customHTTPErrorHandler
-
 	// middleware before routing
 	router.Pre(
+		p.addStartTime(),
 		middleware.RemoveTrailingSlash(),
 		p.stripInvalidHeaders(),
 		p.addUserContext(), // get user information from token before handling request
-
+		//echoprometheus.NewMiddleware("registration_service_metrics"),
```

Comment:
zombie? :ghost:

Reply:
yep, thanks for catching this, removed in [bfdaea8](https://github.com/codeready-toolchain/registration-service/pull/342/commits/bfdaea85ce51c7d826c6005b8d1c478d47c99eb6)